src/utils/media-devices-manager.js

```diff
@@ -20,7 +20,7 @@ export default class MediaDevicesManager {
     this._videoDevices = [];
     this._deviceId = null;
     this._audioTrack = null;
-    this._mediaStream = null;
+    this._mediaStream = new MediaStream();
```

Comment:
I think this should be initialized as null since folks may not have any media streams available and gets assigned later.

Reply:
I've fixed in the next commit https://github.com/mozilla/hubs/pull/3932/commits/64bf34decb577291d4741188497e9ca6e14d7be1